### PR TITLE
Cairo 2.1.1

### LINKED ISSUE #136: Update contract syntax to Cairo v2.1.0

### PR DESCRIPTION
**Pull request by shramee**
resolves #136

Thanks @codeWhizperer for updating the contract exercises!

### CODE REVIEW

**Comment by shramee** [comment]
Continued from #137

File: src/starklings_runner.rs [hunk 1]
```diff
@@ -105,20 +83,40 @@ pub fn run_cairo_program(args: &Args) -> anyhow::Result<String> {
...
             runner.find_function("::main")?,
             &[],
             args.available_gas,
             StarknetState::default(),
         )
         .with_context(|| "Failed to run the function.")?;
     match result.value {
-        cairo_lang_runner::RunResultValue::Success(values) => ret_string
-            .push_str(format!("Run completed successfully, returning {values:?}").as_str()),
+        cairo_lang_runner::RunResultValue::Success(values) => {
```
Comment (enitrat) [inline src/starklings_runner.rs:94]: Have you verified that this works correctly? I think using the `ret_string` variable was required here to get the output of the run 

I tried an my previous solution for `enum2.cairo` is not compiling anymore - but the error message doesn't appear

Comment (shramee) [reply src/starklings_runner.rs:94]: Ohh, I tested with some arbitrary cairo and it seemed to compile/show error. Lemme check enum example.

Comment (shramee) [reply src/starklings_runner.rs:94]: I reverted to old signatures but it was still happening,

This seemed to fix the issue.

image

**Comment by shramee** [comment]
@codeWhizperer few fixes on exercises,
- `starknet1` was solved
- `starknet4` had some parts solved
- `starknet5` don't use `generate_trait` when `interface` is defined.

@enitrat Lemme know if things looks good.

**Comment by barretodavid** [comment]
I'm planning to use Starklings for Basecamp tomorrow. Any chance we can get this PR merged?

**Pull request merged**